### ISSUE TITLE
RFSOC-AMC SI verification

### ISSUE THREAD

**Issue by tprzywoz** [sinara-hw/RFSOC-AMC#97]
PL0   DDR4_2666

Data Write
![image](https://user-images.githubusercontent.com/30114557/122893572-bacb7700-d346-11eb-8472-eda77e5dd353.png)
![image](https://user-images.githubusercontent.com/30114557/122893629-ca4ac000-d346-11eb-9095-20927d8b7e79.png)
![image](https://user-images.githubusercontent.com/30114557/122893668-d3d42800-d346-11eb-80af-0b93b2c3acf0.png)
![image](https://user-images.githubusercontent.com/30114557/122893705-dc2c6300-d346-11eb-8a97-356e5150234d.png)

Data Read
![image](https://user-images.githubusercontent.com/30114557/122895819-bf912a80-d348-11eb-9bbe-ba542096b71c.png)
![image](https://user-images.githubusercontent.com/30114557/122895902-d172cd80-d348-11eb-83ed-7554ad024d35.png)
![image](https://user-images.githubusercontent.com/30114557/122895945-de8fbc80-d348-11eb-89d5-046676127425.png)

Address
![image](https://user-images.githubusercontent.com/30114557/122896045-f9623100-d348-11eb-83ca-29b1a9570a0c.png)
![image](https://user-images.githubusercontent.com/30114557/122896087-02530280-d349-11eb-87dd-9f34a7d584e5.png)
![image](https://user-images.githubusercontent.com/30114557/122896129-0da62e00-d349-11eb-8e96-53d0bc527c99.png)
![image](https://user-images.githubusercontent.com/30114557/122896183-1ac31d00-d349-11eb-8eee-d57c25efedb9.png)
![image](https://user-images.githubusercontent.com/30114557/122896211-21519480-d349-11eb-82d8-40801302910e.png)

Differential Nets
![image](https://user-images.githubusercontent.com/30114557/122896639-8d33fd00-d349-11eb-9141-11b84b4d16e5.png)

CLK-DQS Skew
![image](https://user-images.githubusercontent.com/30114557/122896690-9c1aaf80-d349-11eb-9c55-6a925c7e2da7.png)

**Comment by tprzywoz** [comment]
PL1 DDR4_2666

Data Write
![image](https://user-images.githubusercontent.com/30114557/122902546-d8044380-d34e-11eb-9ed5-d98f1de4d59c.png)
![image](https://user-images.githubusercontent.com/30114557/122902630-ef433100-d34e-11eb-9d56-e974a79df5de.png)
![image](https://user-images.githubusercontent.com/30114557/122902701-fec27a00-d34e-11eb-9733-c06420abe4e7.png)
![image](https://user-images.githubusercontent.com/30114557/122902763-0da92c80-d34f-11eb-8348-a78104df066f.png)

Data Read
![image](https://user-images.githubusercontent.com/30114557/122902950-38938080-d34f-11eb-8915-90de15db214e.png)
![image](https://user-images.githubusercontent.com/30114557/122903012-48ab6000-d34f-11eb-800b-82b975a6ac96.png)
![image](https://user-images.githubusercontent.com/30114557/122903095-5eb92080-d34f-11eb-8d1b-ff27d8d4764b.png)
![image](https://user-images.githubusercontent.com/30114557/122903130-6973b580-d34f-11eb-87c9-f77177fdeb7e.png)

Address
![image](https://user-images.githubusercontent.com/30114557/122903362-9fb13500-d34f-11eb-8bdf-ec84fef22e9e.png)
![image](https://user-images.githubusercontent.com/30114557/122903425-acce2400-d34f-11eb-9766-c5016db3b229.png)
![image](https://user-images.githubusercontent.com/30114557/122903514-c0798a80-d34f-11eb-97a8-2bbd15982949.png)
![image](https://user-images.githubusercontent.com/30114557/122903677-ec950b80-d34f-11eb-877f-97cfd6640bf3.png)
![image](https://user-images.githubusercontent.com/30114557/122903709-f585dd00-d34f-11eb-85fd-bb08d534979e.png)

Differential Nets
![image](https://user-images.githubusercontent.com/30114557/122904007-3bdb3c00-d350-11eb-99e3-cdabd7194c9e.png)

CLK-DQS Skew
![image](https://user-images.githubusercontent.com/30114557/122904111-50b7cf80-d350-11eb-8bfc-2adf0510dafc.png)

**Comment by kaolpr** [comment]
@tprzywoz Have you done simulations for MGT lines?

**Comment by tprzywoz** [comment]
Yes, to AMC edge connector with full speed, to RTM connector 12 Gb/s.

**Comment by gkasprow** [comment]
Is there a chance that this board could work at 28Gbit/s over RTM ? We can use a higher grade RTM connector.

**Comment by tprzywoz** [comment]
Eye diagrams are fine for 25Gbit/s. At 28 Gbit/s there are problems with the last Quad which has mounting options and more vias.

**Comment by gkasprow** [comment]
I assume this is without the RTM board.
Anyway, there is an idea to use Samtec Flyover to QSFP cable. From one side we exit close to MGT, from the other side we have a QSFP28 connector. We only need to find a place for Flyover connector, but it is tiny,
We could implement it in new revision,
